### PR TITLE
Implement global wait for iframe function (close #10)

### LINKED ISSUE #10: Implement wait for iframe function

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @AlexanderMoskovkin

### CODE REVIEW

**Comment by helen-dikareva** [comment]
please check strings @MargaritaLoseva

File: README.md [hunk 1]
```diff
@@ -99,6 +99,31 @@ asyncTest('test with wait', function () {
...
+```
+
+Example:
+```js
+asyncTest('test with wait for iframe', function () {
+    var iframe = document.createElement('iframe');
+
+    iframe.src = window.QUnitGlobals.getResourceUrl('../data/iframe.html');
+
+    window.setTimeout(function () {
```
Comment (AlexanderMoskovkin) [inline README.md]: Let's remove `setTimeout`. I suppose it's a little redundant.

File: README.md [hunk 2]
```diff
@@ -99,6 +99,31 @@ asyncTest('test with wait', function () {
 });
 ```
 
+####Wait for an iframe action
+```js
+window.QUnitGlobals.waitForIframe(iframe);    // returns Promise
+// iframe is 'iframe' element for waiting
```
Comment (MargaritaLoseva) [inline README.md]: iframe is an iframe element to wait for

**Comment by helen-dikareva** [comment]
FPR

**Pull request merged**